paimon-flink/paimon-flink-common/src/main/java/org/apache/paimon/flink/sink/RowDataStoreWriteOperator.java

```diff
@@ -82,6 +91,27 @@ public void setup(
         if (logSinkFunction != null) {
             FunctionUtils.setFunctionRuntimeContext(logSinkFunction, getRuntimeContext());
         }
+        Options options = Options.fromMap(table.options());
```

Comment:
Can these logical just in `PrepareCommitOperator`?

Reply:
Thanks @JingsongLi , DONE